deeposlandia/webapp/templates/predictor.html

```diff
@@ -132,6 +132,20 @@ <h2>Making of</h2>
   </ul>
 </p>
 
+<div class=infoframe>
+  <h4>GPDR compliance</h4>
+  <p>
+    The images are stored on our server when the uploading action is done. We
```

Comment:
Suggestion:

"The uploaded images are stored on our server."

Reply:
Done.